ntoskrnl/inbv/inbv.c

```diff
@@ -1142,7 +1169,6 @@ FinalizeBootLogo(VOID)
     /* Reset progress bar and lock */
 #ifdef INBV_ROTBAR_IMPLEMENTED
     PltRotBarStatus = 3;
-    RotBarThreadActive = FALSE;
```

Comment:
Dunno if it's actually good (nor not) to have this flag set here or inside the thread (as I tried to do in that patch).

Reply:
Ok, let it be inside thread.